app/src/test/java/com/ncnf/map/MapHandlerTest.java

```diff
@@ -11,7 +11,7 @@
 
     @Test
     public void testPositionSetterGetter(){
-        MapHandler mapHandler = new MapHandler(null, null, null, null);
+        MapHandler mapHandler = new MapHandler(null, null, null, null, null);
```

Comment:
Instead of calling a constructor with null arguments, you should create another constructor that doesn't take all those arguments

Reply:
Eloi is right, but since it is in a test it is not very important

Reply:
Yeah that's what I was thinking, it's only for this unit test that we call it with null parameters. I'm not sure it's enough of a reason to create a second constructor